TicTacToeApp/Board.swift

```diff
@@ -0,0 +1,48 @@
+
+public class Board {
```

Comment:
Overall, I like the short methods, use of Swift features like `reduce`, and the small public interface. I would pay a bit more attention to names and try to leverage named parameters more.

The SRP violation mentioned in a different comment is definitely the big thing to fix in this class.